components/Profile/Profile.tsx

```diff
@@ -34,14 +29,7 @@ const Profile: React.FC<ProfileProps> = (props: ProfileProps) => {
       </Text>
       {props.participantType === 'Deelnemer' ? (
         <View style={styles.activityContainer}>
-          <Text
-            style={[
-              styles.activityHeader,
-              { color: Colors[colorScheme].headerColor },
-            ]}
-          >
-            Workshopkeuze
-          </Text>
+          <Text style={[styles.activityHeader]}>Workshopkeuze</Text>
```

Comment:
This isn't using HeaderText so it won't be blue. Confirm if this is what you want :)

Reply:
Kben ni goe bezig haha